modules/seqexec/model/src/main/scala/seqexec/model/enum/Guiding.scala

```diff
@@ -18,10 +18,14 @@ object Guiding {
   val all: List[Guiding] =
     List(Guide, Park, Freeze)
 
-  implicit val equal: Eq[Guiding] =
-    Eq.fromUniversalEquals
-
   def fromString(s: String): Option[Guiding] =
     all.find(_.configValue === s)
 
+  /** @group Typeclass Instances */
+  implicit val GuidingEnumerated: Enumerated[Guiding] =
+    new Enumerated[Guiding] {
+      def all = Guiding.all
```

Comment:
This is a minor complain, but I would prefer if the different `Enumerated` followed the same implementation style. Right now there are some that define `all` directly as a `List` of all posible values, and other implementations that define the list in the container object, and have the `Enumerated.all` refer to that one.

Reply:
Fixed, we now have `val all: List[A]` in all enum types